test/Microsoft.IdentityModel.TestUtils/TokenValidationExtensibility/Tests/IssuerExtensibilityTestCases.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using Xunit;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.IdentityModel.Logging;
+
+namespace Microsoft.IdentityModel.TestUtils.TokenValidationExtensibility.Tests
+{
+    public partial class ExtensibilityTesting
+    {
+        public static TheoryData<IssuerExtensibilityTheoryData> GenerateIssuerExtensibilityTestCases(
+            string tokenHandlerType,
+            int extraStackFrames,
+            string stackFrameFileName)
```

Comment:
Can't you automatically capture this similarly to how the `ValidationError` does it?
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/ae57a3cbf9bcef8a1f0789010bf19cb32035d6ab/src/Microsoft.IdentityModel.Tokens/Validation/Results/Details/ValidationError.cs#L237

Reply:
We can capture the file, but the file we need to compare against is not the one that calls this method, so in this situation I think it is best to keep it as is.